dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPI.java

```diff
@@ -114,4 +116,21 @@ Experiment deleteVariant(String experimentId, String variantName, User user)
 
     Experiment deleteTargetingCondition(String experimentId, String conditionId, User user)
             throws DotDataException, DotSecurityException;
+
+    /**
+     * Return a list of the current RUNNING Experiments.
+     *
+     * @return
+     */
+    List<Experiment> getExperimentRunning() throws DotDataException;
```

Comment:
rename to "getRunningExperiments"

Reply:
done https://github.com/dotCMS/core/pull/23334/commits/5c94901d63ceb1249a7832d916be5be6f3b380dd